app/presenters/concerns/pagination_api_data.rb

```diff
@@ -0,0 +1,60 @@
+module PaginationAPIData
+  include Pagy::Backend
+
+  DEFAULT_PER_PAGE = 50
+  MAX_PER_PAGE = 50
+
+  def serialized_applications_data
+    %({"data":[#{serialized_applications.join(',')}], "links": #{links.to_json}, "meta": #{VendorAPI::MetaPresenter.new(active_version, @pagy&.count).as_json}})
+  end
+
+  def links
+    links_hash = {
+      first: @pagy_meta[:first_url] || options[:url],
+      last: @pagy_meta[:last_url] || options[:url],
+      self: @pagy_meta[:page_url] || options[:url],
+    }
+    links_hash[:prev] = @pagy_meta[:prev_url] if !@pagy_meta[:prev].nil?
+    links_hash[:next] = @pagy_meta[:next_url] if !@pagy_meta[:next].nil?
```

Comment:
@cwrw unsure on this one... if they don't use pagination should we also include the prev and next attributes in the links object?